src/MICore/LaunchOptions.cs

```diff
@@ -597,7 +597,12 @@ public LaunchCompleteCommand LaunchCompleteCommand
             }
         }
 
-        public static LaunchOptions GetInstance(HostConfigurationStore configStore, string exePath, string args, string dir, string options, IDeviceAppLauncherEventCallback eventCallback, TargetEngine targetEngine, Logger logger)
+        public static LaunchOptions GetInstance(HostConfigurationStore configStore, string exePath, string args, string dir, string options, IDeviceAppLauncherEventCallback eventCallback, TargetEngine targetEngine)
+        {
+            return GetInstance(configStore, exePath, args, dir, options, eventCallback, targetEngine, null);
+        }
+    
+        public static LaunchOptions GetInstance(HostConfigurationStore configStore, string exePath, string args, string dir, string options, IDeviceAppLauncherEventCallback eventCallback, TargetEngine targetEngine, Logger logger = null)
```

Comment:
Do we want to still provide a default value for `logger`, given that you're adding another method without that parameter?